T3/Gui/Graph/Interaction/SymbolBrowser.cs

```diff
@@ -51,20 +52,19 @@ public void Draw()
         {
             if (!IsOpen)
             {
-                if (!ImGui.IsWindowFocused() || !ImGui.IsKeyReleased((ImGuiKey)Key.Tab))
+                if (!ImGui.IsWindowFocused() || !ImGui.IsKeyReleased((ImGuiKey)Key.Tab) || !ImGui.IsWindowHovered())
```

Comment:
`|| !ImGui.IsWindowHovered()`

This sounds scary. Did you test this thoroughly for side effects?

Reply:
I did test it, but i cant say i put it through the wringer. I'm not married to this idea, it just seemed strange that if my mouse is off-screen, i can press Tab and Enter and I can find out 10 minutes later I made an Execute node way off screen